drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
-	callback    driverapi.DriverCallback
```

Comment:
It is nonsense to have a singleton remote driver which isn't actually a driver; that, in large part, is what motivates this pull request. Even if there were one, how would that help -- any plugin integration code would need access to unexported fields or methods here.

The only reasonable choice, in the design given by #127, is to do the integration in `Init()`. Possibly that integration will explicitly store the callback for later, but it could also just close over it in a handler or callback of its own.